aboutpage.html

```diff
@@ -0,0 +1,146 @@
+        <hr>
+        <section>
+            <h2 class="event-concert">Discovers previous piano concert</h2>
+            <div class="artist-bottom-line"></div>
+            <p class="to-move">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatum sint quas fuga dignissimos expedita impedit possimus deserunt hic in facere?
+                <br><br> Lorem ipsum dolor, sit amet consectetur adipisicing elit. Repellendus sapiente facilis beatae praesentium quia maiores! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nisi nesciunt et delectus, doloremque minus perspiciatis.
+            </p>
+            <div class="concert-photo">
+                <ul class="for-2022">
+                    <li class="for-2023"> <img src="/images/2022 (1).jpeg " alt="2022 concert "></li>
+                    <li class="for-desk-img"><img src="/images/2022 (2).jpeg " alt="2023 concert "></li>
+                </ul>
+            </div>
+            <div class="photo-sec">
+                <ul>
+                    <li class="for-summer"> <img src="/images/2022 (1).jpeg " alt="2022 concert "></li>
+                    <li class="for-2019"><img src="/images/2022 (2).jpeg " alt="2023 concert "></li>
+                </ul>
+            </div>
+        </section>
+        <section class="for-partner-sec">
+            <h3 class="partner">Partners</h3>
+            <div class="artist-bottom-line"></div>
+            <div class="for-width">
+                <ul class="burn" id="mozilla">
+                    <li class="color-burn"><img src="images/mozilla.png" alt="partner-logo "></li>
+                    <li class="color-burn"><img src="images/color-burn.png" alt="partner-logo "></li>
+                    <li class="color-burn"><img src="images/rays.png " alt="partner-logo "></li>
+                    <li class="color-burn"><img src="images/studio.png" alt="partner-logo "></li>
+                    <li class="color-burn"><img src="images/piano logo.png" alt="partner-logo "></li>
+                </ul>
+            </div>
+        </section>
+        <section class="final-card-sec">
+            <ul class="for-footer-sec">
+                <li><img src="images/piano logo.png" alt="piano-logo" class="music"></li>
+                <li class="move-display">
+                    <p>2023 Creative Common Myanmar Center,somes right reserved<br> 사랑하는 친구에게,오랜만에 펜을 들어 글을 쓰는 순간, 당신이 떠올랐습니다. 얼마나 그리웠던지 몰라요</p>
+
+                </li>
+
+            </ul>
+        </section>
+        <section class="card-sec">
+
+            <ul class="for-footer">
+                <li><img src="images/piano logo.png" alt="piano-logo" class="footer-music"></li>
+                <li class="rights">
+                    <p>2023 Creative Common Myanmar Center,somes right reserved</p>
+
+                </li>
+
+
+        </section>
+        <script src=" /index.js "></script>
+        </body>

```

Comment:
- [ ] Please work on the About page too. Ask pointed out above, we are still having the same issues.

| Your Implimentation | The Design |
| --- | --- |
| ![Web capture_12-7-2023_133946_lameck1-scaling-space-fiesta-7pr77wr9pgj3vp4-5503 preview app github de](https://github.com/Hsuwaddyhtat22/musical-giggle-capstone/assets/45002654/4dc9727e-ccef-4553-908a-0f546e709f4b) | ![image](https://github.com/Hsuwaddyhtat22/musical-giggle-capstone/assets/45002654/1bc27db1-d4f4-4f72-a6e3-507719f959e6) |

On this page, please work on the following:

- Add that page margin.
- Elements on the about are centered on the page, even texts are center-aligned. Please consider adjusting your implementation to align with the provided design.